package.json

```diff
@@ -5,7 +5,7 @@
     "scripts": {
         "typecheck": "tsc --noEmit",
         "lint": "if [ \"$npm_config_fix\" ]; then eslint . --ext .js,.ts --cache --fix; else eslint . --ext .js,.ts --cache; fi",
-        "prepare": "husky install",
+        "prepare": "[ -f .git/hooks ] && husky install || true",
```

Comment:
Can you explain?

Reply:
yes I can, I was running into issues where it was trying to do the husky install from this prepare step when you were installing azle into an example or tests. and it was failing because it didn't have husky installed. I was exploring some different options. It seems like the easiest one was to just not do the prepare step if .git/hooks doesn't exists. I'm guessing that the fact that this happened all of a sudden hints at a deeper problem and we should probably talk about it now that you are back from your vacation

Reply:
Make sure that top level azle is installed before any other npm installs are called

Reply:
it looks like it was just a network error that interrupted the npm install at the top level